Documents/UseCasesForWebsite.md

```diff
@@ -0,0 +1,39 @@
+    maybe a telephone number and the visitor can choose by a select list what the visitor wants to donate. 
+- **goto the public gallery page** which can also be found with a link on the menu of the site. The images will be sorted with the latest one first. 
+- **View info page** by following the link on the menu of the website. On the info page the visitor can read information about the mission of this project and how it�s started, 
+   success stories and comments of the artist involved. 
+- **Like a picture**. When a visitor likes a picture, he/she can click on a like button. 
+- **View artwork**. When the visitor goes to the public gallery by clicking on the gallery link. When the visitor then clicks on a image shown in the gallery a modal screen is shown zoomed 
+  in at the whole screen where the image is also shown together with the name of the artist and the email address of the artist so the visitor can contact the artist who made that piece. 
+  Also  around 3 thumbnails of non-prosthetics) are shown , but also a reference to general other artwork
+
+**Artist** - The same as a visitor but he difference is that this person makes proffesinal art. 
+An artist could use this website for the same reasons as a regular user. Also the artist can fill in a form and 
+send two pictures of art the artist make so the admin can look f the artist is good enough to paint a prosthetic for a amputee or for the fun raising. 
+-  a artist **can apply to be a part of the project** by filling in a apply form with the name and email address , send in X images of work he/she made and gets approved or disapproved. 
+   The admin then gets a email that a artist has applied. 
+
+**Admin** - A person who is in control of this project and the website
+An admin would use this website to manage the open gallery, accept  or decline incoming pictures that people want to post to the promotional gallery (kind of a moderator role), and they could also manage artists as well.
+A admin can 
+-  **give an artist wants has applied to become a "Professional Artist" on the site/app the right to become a  "Professional Artist".**
+- **Managing request from a amputee** means that  the admin sees the request and the admin can choose to approve the request.
+- **View artist submissions.** When a artist sends in the apply form, there must be 2 pictures of art the artist made. These are stored in a database. 
+  On the private gallery all send in images are shown sorted by the latest first. The status is automatically set to send-in. 
+- **Approve  artist.** The admin sets then the status to approved by or using a button or a select list. The artist then gets a email that he/she is approved. 
+- **Disapprove artist.**  The admin sets then the status to disapproved by or using a button or a select list. Also here the artist will receive a email that he/she is disapproved
+ 
+**System** � Notify old pending requests
```

Comment:
System description should be improved. This description is actually what a use case name should havfe been.